src/components/RadioButton/RadioButton.st.css

```diff
@@ -83,6 +83,10 @@
     border: 1px solid applyOpacity(value(CurrentBorderColor), 0.6);
 }
 
+.withChildren .radioIcon {
+    top:calc(50% - 8px);
+}
+
```

Comment:
Where did all the rest of the style changes go 🤔

Reply:
after the changes to the radio button style here (https://github.com/wix/wix-ui-tpa/commit/5544eb0ad083562a5edd62211397c8296ce32d60#diff-0eb1fa28aba0ffad138a3643fee270095088f5016442d1bf48252875d0873758) some of them became irrelevant :)